visualisation/nzvm.py

```diff
@@ -130,14 +135,34 @@ def main():
     if length != nx*ny*nz:
         print("Error: Inconsistent VM size. Check model and nx, ny, nz parameters and try again. Expected",nx*ny*nz,"values, got",len(array))
         quit()
+
+    if args.perturbation:
+        pert_length, pert_array = load_data(args.perturbation, nx, ny, nz)
+        if pert_length != nx*ny*nz:
+            print("Error: Inconsistent perturbation file size. Check model and nx, ny, nz parameters and try again. Expected",nx*ny*nz,"values, got",len(pert_array))
+            quit()
     
     if axis == 'x':
         data = array[start_point[1],:,start_point[0]:(start_point[0]+delta+1)] * KM_TO_M
     else:
         data = array[start_point[1]:(start_point[1]+delta+1),:,start_point[0]] * KM_TO_M
         data = np.swapaxes(data,0,1)
-        
-    display_data_matplot(data, int(len(data)/nz), nz, args.maximum, hh)
+    
+    if args.perturbation:
+        fig, axs = plt.subplots(2, 1, constrained_layout=True)
+        display_data_matplot(data, args.maximum, hh, axs[0], "Cross section (without perturbation)")
```

Comment:
I think this does the same as L165. The extra comment isn't necessary considering that the other title addresses it.

Reply:
There is a subtle difference where `plt.subplots` returns a single subplot (not in an array) when the dimension is (1, 1), which would make the part `axs[0]` cause an error

Reply:
squeeze seems like the argument that fixes this problem.